ScoutSuite/providers/aws/services/iam.py

```diff
@@ -210,30 +210,24 @@ def fetch_password_policy(self, credentials):
 
         except ClientError as e:
             if e.response['Error']['Code'] == 'NoSuchEntity':
-                self.password_policy = {}
-                self.password_policy[
-                    'MinimumPasswordLength'] = '1'  # As of 10/10/2016, 1-character passwords were authorized when no policy exists, even though the console displays 6
-                self.password_policy['RequireUppercaseCharacters'] = False
-                self.password_policy['RequireLowercaseCharacters'] = False
-                self.password_policy['RequireNumbers'] = False
-                self.password_policy['RequireSymbols'] = False
-                self.password_policy['PasswordReusePrevention'] = False
-                self.password_policy['ExpirePasswords'] = False
+                self.password_policy = {'MinimumPasswordLength': '1', 'RequireUppercaseCharacters': False,
+                                        'RequireLowercaseCharacters': False, 'RequireNumbers': False,
+                                        'RequireSymbols': False, 'PasswordReusePrevention': False,
+                                        'ExpirePasswords': False}
```

Comment:
Same here, I would keep it as it was. Or if we ever decide to change all the services, I would at least put the different entries on new lines as it is much more readable.

Reply:
Same as before, but I would agree that it would be better to split it 1-line per arguments